ms2pip/ms2pipC.py

```diff
@@ -692,6 +752,10 @@ def __init__(
         # Validate requested model
         if self.model in MODELS.keys():
             logger.info("using %s models", self.model)
+            if "xgboost_model_files" in MODELS[self.model].keys():
+                for _, model_file in MODELS[self.model]["xgboost_model_files"].items():
+                    if not check_model_presence(model_file, MODELS[self.model]["model_hash"][model_file]):
+                        download_model(model_file, MODELS[self.model]["model_hash"][model_file])
```

Comment:
Why don't we just package the models like we do with the C-models?

Reply:
they are way to big to put in the package